site/public/js/docker_interface.js

```diff
@@ -31,6 +31,7 @@ function filterOnClick() {
         const this_row = $(this);
         let hide = true;
         $(this).find('.badge').each(function () {
+            // eslint-disable-next-line eqeqeq
             if ($(this).text() == this_filter) {
```

Comment:
In the jQuery documentations, `.data` can return any type.  However, in my testing, it looks like in this function (`filterOnClick`), `.data` is only returning a `string`, the selected filter's name.